api/NetworkInformation.json

```diff
@@ -15,10 +15,18 @@
             "version_added": "79"
           },
           "firefox": {
-            "version_added": false
+            "version_added": "31",
```

Comment:
FYI, this was an omission as far as I can tell. I have added versions for desktop behind flag for anything defined in Android.